packages/gatsby-source-shopify/README.md

```diff
@@ -70,6 +70,29 @@ plugins: [
       // Possible values are: 'shop' and 'content'.
       // Defaults to ['shop', 'content'].
       includeCollections: ["shop", "content"],
+
+      // Allow overriding the default queries
+      // This allows you to include/exclude extra fields when sourcing nodes
+      // Available keys are: articles, blogs, collections, products, shopPolicies, and pages
+      // Queries need to accept arguments for first and after
+      // Defaults can be found in ./src/queries.js
+      queries: {
```

Comment:
I think this config documentation needs to be updated to shopifyQueries in order for it to work.